cli/cmd/delete.go

```diff
@@ -32,7 +32,16 @@ var deleteCmd = &cobra.Command{
 	Short:        "Delete an existing stack",
 	Long:         "Delete the stack with the given name.",
 	SilenceUsage: true,
-	RunE:         runDelete,
+	RunE: func(cmd *cobra.Command, args []string) error {
+		fmt.Println("Running delete")
```

Comment:
Please change to our standard logging. Use of fmt.Print* is frowned upon.

Reply:
Updated to using logrus.